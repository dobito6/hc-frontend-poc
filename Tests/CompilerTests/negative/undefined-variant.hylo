//! no-std

fun f(x: auto Void) auto {
  let { ((), ()) }
}

public fun main() {
  let a = f@let(x: ())
  let b = f@sink(x: ())
}
