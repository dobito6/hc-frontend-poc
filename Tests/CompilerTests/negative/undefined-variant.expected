negative/undefined-variant.hylo:9.11-17: error: undefined symbol 'f@sink'
  let b = f@sink(x: ())
          ~~~~~~
